metrictank.go

```diff
@@ -102,6 +102,11 @@ func init() {
 func main() {
 	startupTime = time.Now()
 
+	/***********************************
+		Initialize Logger
+	***********************************/
+	log.NewLogger(0, "console", fmt.Sprintf(`{"level": %d, "formatting":false}`, logLevel))
```

Comment:
doesn't this have to come after the flag.Parse call though?

Reply:
It definitely should if the logLevel specified mattered. But as you said in the commented issue #4055, it doesn't actually affect the log levels until they're set later on.

I agree, it should probably go after the version check.